src/Gui/Language/Translator.cpp

```diff
@@ -251,6 +251,19 @@ std::string Translator::locale(const std::string& lang) const
     return loc;
 }
 
+bool Translator::setLocale(const std::string& language) const
+{
+    auto loc = QLocale::c(); //Defaulting to POSIX locale
+    auto bcp47 = Translator::instance()->locale(language);
```

Comment:
The call of `Translator::instance()` should be dropped as calling the function setLocale() is done for a valid Translator object.

Reply:
Indeed. Copy-paste syndrome. Thanks for the catch. Will update soon.